src/Explorer/QueryCopilot/QueryCopilotPromptbar.tsx

```diff
@@ -301,7 +303,38 @@ export const QueryCopilotPromptbar: React.FC<QueryCopilotPromptProps> = ({
       return "Content is updated";
     }
   };
+  const openSamplePrompts = () => {
+    inputEdited.current = true;
+    setShowSamplePrompts(true);
+  };
+  const totalSuggestions = useMemo(
+    () => [...filteredSuggestedPrompts, ...filteredHistories],
+    [filteredSuggestedPrompts, filteredHistories],
+  );
+
+  const handleKeyDownForInput = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === "ArrowDown") {
```

Comment:
Maybe let's use existing `NormalizedEventKey` from https://github.com/Azure/cosmos-explorer/blob/master/src/Common/Constants.ts#L379 instead of strings.